generate-indexes/src/test/scala/nl/lumc/sasc/biopet/pipelines/generateindexes/DownloadGenomesTest.scala

```diff
@@ -45,7 +45,7 @@ class DownloadGenomesTest extends TestNGSuite with Matchers {
   @Test
   def testNcbiAssembly(): Unit = {
     val pipeline = initPipeline(Map())
-    pipeline.referenceConfig = Map("s1" -> Map("g1" -> Map("ncbi_assembly_id" -> "id")))
+    pipeline.referenceConfig = Map("s1" -> Map("g1" -> Map("ncbi_assembly_report" -> "id")))
```

Comment:
This is now called 'report' in here and tools, but still 'assembly id' in the tool extension. To be fully consistent, I think we should change that there as well.